src/Hedgehog/Gen.fs

```diff
@@ -493,26 +493,22 @@ module Gen =
         return System.Guid bs
     }
 
-    /// Generates a random instant in time expressed as a date and time of day.
+    /// Generates a random instant in time expressed as a date and time of day
+    /// including millisecond part. Two different instants can be represented
+    /// by the same string if the millisecond part isn't included
+    /// in the string.
```

Comment:
Please check whether the above XML document is expressed naturally.

Reply:
I would leave the comment as it was

```
/// Generates a random instant in time expressed as a date and time of day.
```

unless we decide to add two different versions of `Gen.dateTime`, one that includes the millisecond and one that doesn't.

However, I think we should have only one, and that's the one you're proposing in this PR, so I think the comment should be the original one (because we're fixing a bug, after all).